### ISSUE TITLE
Add first-class support for configuring tmpfs

### ISSUE THREAD

**Issue by vpavic** [testcontainers/testcontainers-java#673]
Since docker-java added support for tmpfs in `3.1.0-rc-3` (see docker-java/docker-java#1022), it is now possible to configure TestContainers to use tmpfs using something like:

```java
withCreateContainerCmdModifier(cmd -> cmd.getHostConfig().withTmpFs(
		Collections.singletonMap(System.getProperty("java.io.tmpdir"), "rw")));
```

It would be nice if TestContainers could enhance it's API to provide a way to configure this a bit simpler.

**Comment by bsideup** [comment]
Hi @vpavic,

That's indeed something we could add 👍 

FYI @rnorth was experimenting with TmpFS but the results were not that impressive back then if I remember correctly.

**Comment by mbaechler** [comment]
We use TmpFs for some time now on Apache James project and the gain is huge.

**Comment by rnorth** [comment]
Let's give it a go again! It's not hard to do, so fair to add it for teams that will benefit.

**Comment by malltshik** [comment]
How about this solution? Is it okay? 
`new GenericContainer().withTmpFs(singletonMap("/path", "rw")`

**Comment by rnorth** [comment]
Released in [1.10.4](https://github.com/testcontainers/testcontainers-java/releases/tag/1.10.4)!

Thanks for the contribution!
